.travis.yml

```diff
@@ -5,8 +5,12 @@ language: cpp
 env:
   - ARMADILLO=latest
   - ARMADILLO=minimum
+  - SANITY=history
```

Comment:
We have to use `SANITY=history ARMADILLO=latest` here, to tell travis to test against the latest version.

Reply:
In that case, we'd be doing a lot of duplicate testing. The sanity check, in my opinion, should perform that test.

Reply:
I agree about the duplicity of tests. 

As this is now an environment variable, we could potentially _stop_ the CI from running after the history check ran with `exit 0`.  Thus, we wouldn't need to set `ARMADILLO`.

Thoughts?

Reply:
Fine with me, no need to run the same configuration twice.

Reply:
@zoq addressed by e28c0c0e4d8f08052d79f7de4bfb53fe7fce07f5

Reply:
Nice, looks like the other runs don't know the `SANITY` env and fail because of that.

Reply:
@zoq resolved with 882ecf3122d4d0e926f265fabf78733f1a2da4ba and 321f7201d6375f76605d910798841d1a25083b73. 😄